homeassistant/components/uptime/sensor.py

```diff
@@ -26,7 +30,12 @@
 )
 
 
-async def async_setup_platform(hass, config, async_add_entities, discovery_info=None):
+async def async_setup_platform(
+    hass: HomeAssistant,
+    config: ConfigType,
+    async_add_entities: AddEntitiesCallback,
+    discovery_info: DiscoveryInfoType | None = None,
+) -> None:
     """Set up the uptime sensor platform."""
     name = config.get(CONF_NAME)
```

Comment:
There's a default name set in the config schema.
```suggestion
    name = config[CONF_NAME]
```